docs/reST/ref/surface.rst

```diff
@@ -130,6 +130,24 @@
 
       .. ## Surface.blit ##
 
+   .. method:: rblit
```

Comment:
Needs lack of keyword arguments documented.

Reply:
This i think i resolved by expanding the docs.